### PR TITLE
Update FBbt mappings

### PR DESCRIPTION
**Pull request by gouttegd**
Manual update of the FBbt mappings (this is not automated yet).

This update brings small corrections to the mappings – the kind of corrections that does not require changing the _terms_ that are mapped, on either side; only the mappings themselves are changed.

### CODE REVIEW

**Comment by shawntanzk** [comment]
I might not be the best person to review this - don't understand any of mapping things >.< have put a issue to ask for a tutorial on that already heh

**Comment by gouttegd** [comment]
To those I asked to review: The changes here were discussed on FBbt (FlyBase/drosophila-developmental-anatomy-ontology#13460, this PR merely conveys the changes done in FBbt so they are reflected in Uberon.

In the future, getting the most recent mappings from FBbt will be done automatically (probably as part of the release process), but we’re waiting for support from the ODK to do that. In the meantime, it has to be a manual process, hence this PR.

**Comment by anitacaron** [comment]
There're more diffs without comments on the referred PR that I cannot check.

File: src/ontology/mappings/fbbt-mappings.sssom.tsv [hunk 1]
```diff
@@ -234,15 +229,13 @@ FBbt:00004850	phallus	skos:exactMatch	UBERON:0008811	HumanCurated
 FBbt:00004856	organ system	skos:exactMatch	UBERON:0000467	HumanCurated
 FBbt:00004857	reproductive system	skos:exactMatch	UBERON:0000990	HumanCurated
 FBbt:00004858	gonad	skos:exactMatch	UBERON:0000991	HumanCurated
-FBbt:00004859	gonadal sheath	skos:exactMatch	UBERON:0004909	HumanCurated
```
[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/mappings/fbbt-mappings.sssom.tsv:237]: I later removed the mapping between FBbt’s `gonadal sheath epithelium` and Uberon’s `epithelium of gonad`, at the request of @dosumis ([see here](https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/pull/1346#discussion_r800431576)). The Uberon term may need to be revised (the text definition and the logical definition don’t match) before a correct mapping can be restored, but this is out of scope for this PR.

File: src/ontology/mappings/fbbt-mappings.sssom.tsv [hunk 2]
```diff
@@ -291,34 +283,31 @@ FBbt:00005063	hemocyte	skos:exactMatch	CL:0000387	HumanCurated
...
 FBbt:00005124	sensory neuron	skos:exactMatch	CL:0000101	HumanCurated
 FBbt:00005125	interneuron	skos:exactMatch	CL:0000099	HumanCurated
-FBbt:00005127	amacrine neuron	skos:exactMatch	CL:0000561	HumanCurated
 FBbt:00005128	pioneer neuron	skos:exactMatch	CL:0000116	HumanCurated
 FBbt:00005130	neurosecretory neuron	skos:exactMatch	CL:0000381	HumanCurated
 FBbt:00005133	serotonergic neuron	skos:exactMatch	CL:0000403	HumanCurated
 FBbt:00005136	sensory nerve	skos:exactMatch	UBERON:0001027	HumanCurated
-FBbt:00005137	ganglion	skos:exactMatch	UBERON:0000045	HumanCurated
 FBbt:00005139	neuropil	skos:exactMatch	UBERON:0002606	HumanCurated
-FBbt:00005140	neuromere	skos:exactMatch	UBERON:0004732	HumanCurated
```
[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/mappings/fbbt-mappings.sssom.tsv:318]: Same as above, see David’s [comment](https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/pull/1346#discussion_r800426237).

File: src/ontology/mappings/fbbt-mappings.sssom.tsv (hunk 1 above)
[the remark this answers is not in the document]
Comment (anitacaron) [reply src/ontology/mappings/fbbt-mappings.sssom.tsv:237]: Alright!

**Pull request merged**